### PR TITLE
add dragdrop for tr

### PR DESCRIPTION
**Pull request by nodephp**
为tr添加drag选项

### CODE REVIEW

File: examples/routers/table.vue [hunk 1]
```diff
@@ -62,6 +62,9 @@
         methods: {
             handleClearCurrentRow () {
                 this.$refs.currentRowTable.clearCurrentRow();
+            },
+            onDragDrop(a,b){
+                console.log(a,b)
```
Comment (wbgneto) [inline examples/routers/table.vue]: ?

Comment (icarusion) [reply examples/routers/table.vue]: in examples catalog, can use `console` to test

**Comment by lipuyu** [comment]
good job

File: examples/routers/table.vue (hunk 1 above)
Comment (dqiang) [reply examples/routers/table.vue]: console.log(typeof a) => "string"
console.log(typeof b)  => "number"
a and b belong to different types

Comment (dqiang) [reply examples/routers/table.vue]: > in examples catalog, can use `console` to test

console.log(typeof a) => "string"
console.log(typeof b) => "number"
a and b belong to different types

**Comment by wenha** [comment]
I've added "draggable" and "on-drag-drop" ,but it didn't work, my iview version is 3.4.2

**Pull request merged**